src/main/java/org/klodnicki/service/BookService.java

```diff
@@ -39,4 +40,23 @@ public void update(BookInfo bookInfo) {
     public List<BookInfo> findBooksByTitleAndAuthor(String title, String author) {
         return bookRepository.findBooksByTitleAndAuthor(title, author);
     }
+
+    public List<String> showBooks() throws NotFoundInDatabaseException {
```

Comment:
This method isn't needed. You can just use prepareListOfAllBooks() by making it public instead of private.

Reply:
OK done!

Reply:
This is not solved :)
Please tell me if you need this method? Can't you just directly use `prepareListOfAllBooks`? This method (`showBooks`) doesn't have proper name as it doesn't "show" books.

Reply:
Im sorry I forgot to remove it.